core/templates/pages/topics-and-skills-dashboard-page/modals/unassign-skill-from-topics-modal.component.ts

```diff
@@ -42,52 +43,111 @@ export class UnassignSkillFromTopicsModalComponent
   // and we need to do non-null assertion. For more information, see
   // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
   skillId!: string;
-  topicsAssignments!: TopicAssignments;
+  eligibleTopicNameToTopicAssignments!: TopicNameToTopicAssignments;
+  nonEligibleTopicNameToTopicAssignments !: TopicNameToTopicAssignments;
```

Comment:
Use "ineligible" throughout, instead of "non eligible".

Could you also please add a comment above line 46 explaining what "eligible" means?

Reply:
Updated nonEligible to inEligible and also added a comment before eligible.